github-bugzilla-content.js

```diff
@@ -41,6 +41,14 @@ function getPRTitle() {
 }
 
 
+/**
+  * Retrieve the PR url
+  */
+function getPRUrl() {
+    return document.URL;
+}
```

Comment:
I tossed this in a function because I didn't know if we might want to remove parts of the url at some point. I don't know what all possible PR urls look like.